src/main/java/com/heatheryou/httpserver/Main.java

```diff
@@ -4,31 +4,27 @@
 import com.heatheryou.httpserver.route.handler.BuildResponse;
 import com.heatheryou.httpserver.route.handler.ResponseBuilder;
 
-import java.io.IOException;
-import java.util.Arrays;
-
 public class Main {
-    public static void main(String[] args) throws IOException {
-        if (args.length != 1) {
-            System.out.println(Arrays.toString(args));
-            System.err.println("Usage: java HTTPServer <port number>");
-            System.exit(1);
+    public static void main(String[] args) throws Exception {
+        ISystemOutput systemOutput;
+        systemOutput = new SystemOutput();
+
+        if (CommandLineArgs.isInvalid(args)) {
+            CommandLineArgs.displayErr(systemOutput);
```

Comment:
Maybe a `Setup` or `Bootstrap` or something class that would take both the `systemOutput` and the `CommandLineArgs`?

```
Setup setup = new Setup(new CommandLineArgs(args), new SystemOut());
int port = setup.execute();  // Maybe later, if you needed it, return a Config and do something like config.getPort();
```

I think this conditional might end up under test then, and it may also let you pull the `displayErr` method out of the `CommandLineArgs`.  It might end up in the `Setup` class.

This is just an idea.  It could work out, or could end up being difficult.